cmd/influxd/inspect/report_tsi1.go

```diff
@@ -0,0 +1,104 @@
+package inspect
+
+import (
+	"errors"
+	"io"
+	"os"
+	"path"
+
+	"github.com/influxdata/influxdb"
+
+	"github.com/influxdata/influxdb/logger"
+	"github.com/influxdata/influxdb/tsdb/tsi1"
+	"github.com/spf13/cobra"
+	"go.uber.org/zap/zapcore"
+)
+
+// Command represents the program execution for "influxd inspect report-tsi".
+var reportTSIFlags = struct {
+	// Standard input/output, overridden for testing.
+	Stderr io.Writer
+	Stdout io.Writer
+
+	// Data path options
+	Path           string // optional. Defaults to dbPath/engine/index
+	SeriesFilePath string // optional. Defaults to dbPath/_series
+
+	// Tenant filtering options
+	Org    string
+	Bucket string
+
+	// Reporting options
+	TopN          int
+	ByMeasurement bool
+	byTagKey      bool // currently unused
+}{}
+
+// NewReportTsiCommand returns a new instance of Command with default setting applied.
+func NewReportTSICommand() *cobra.Command {
+	reportTSICommand := &cobra.Command{
+		Use:   "report-tsi",
+		Short: "Reports the cardinality of tsi files short",
+		Long:  `Reports the cardinality of tsi files long.`,
+		RunE:  RunReportTSI,
+	}
```

Comment:
I know it's like this in some other places but I find it easier if you just shorten the variable name to `cmd`.